build.cmd

```diff
@@ -178,6 +178,9 @@ REM === Start the build steps
 REM ===
 REM =========================================================================================
 
+echo %__MsgPrefix%Using environment: "%__VSToolsRoot%\VsDevCmd.bat"
```

Comment:
Blindly running it inside run.cmd screws up arm64 environment for either cross or native build respectively. So, setting it upfront avoids running it again inside run.cmd while we can run private arm64 private build set up that follows.